code/style.css

```diff
@@ -1,4 +1,222 @@
+*{
+  margin: 0;
+  padding: 0;
```

Comment:
Why did you add a 0 padding?

Reply:
This way I override the default margin and padding formatting that the elements in my HTML might have. I read it is a good practice to do in order to avoid some mysterious default margins or padding popping up on some elements.